Sources/Holocron/Keychain+StorageProvider.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Keychain+StorageProvider.swift
+//  Holocron-iOS
+//
+//  Created by Pranjal Satija on 4/10/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import KeychainAccess
+
+public extension StorageProvider {
+    typealias Keychain = KeychainAccess.Keychain
+}
+
+extension Keychain: StorageProvider {
```

Comment:
This extension can be marked `public` and each function does not have to be specified as `public`.

Reply:
Can't do that. Extensions that declare protocol conformances can't be `public`.